OptionSelector/CategoryModel.m

```diff
@@ -0,0 +1,52 @@
+//
+//  CategoryModel.m
+//  OptionSelector
+//
+//  Created by Henna on 8/10/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "CategoryModel.h"
+#import "CQCategory.h"
+
+@implementation CategoryModel
+
++ (CategoryModel *)sharedInstance
+{
+    static dispatch_once_t once;
+    static id sharedInstance;
+    dispatch_once(&once, ^{
+        sharedInstance = [[self alloc] init];
```

Comment:
This is great! Haven't seen anyone else use a singleton yet. The only thing that's missing is that you should probably just call `initializeModel here after you create the shared instance so you don't ever have to call it externally.